File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.size()!=t.size()) return false ;
        vector<int> store(26);
        for (char x : s ){
            int index = x - 'a';
            store[index]++ ;
        }
        for (char x : t){
            int index = x - 'a';
            if (store[index]<=0) return false ;
            store[index]--;
        }
        return true ;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False  
        a = [0] * 26
        for x in s:
            idx = ord(x) - ord('a')
            a[idx] += 1
        for x in t:
            idx = ord(x) - ord('a')
            if a[idx] == 0:
                return False
            a[idx] -= 1
        return True